setup-local/src/artifacts.js

```diff
@@ -0,0 +1,27 @@
+const artifact = require('@actions/artifact');
+const core = require('@actions/core');
+
+class ArtifactsManager {
+  /**
+   * Upload artifacts to GitHub workflow
+   * @param {String} artifactName Name by which the artifact should be available post uploading
+   * @param {String[]} files Files to upload
+   * @param {String} rootFolder Folder in which the files reside
+   * @returns {Promise<artifact.UploadResponse>} Response of the upload operation
+   */
+  static async uploadArtifacts(artifactName, files, rootFolder) {
```

Comment:
in the test cases, rootFolder is an array but otherwise it's a simple path. let's make them same?

Reply:
fixed that.

Reply:
resolving this.